src/easydmp/auth/lib.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import login
+
+from easydmp.eventlog.utils import log_event
+
+
+def su_login(request, user):
+    original_user = request.user
```

Comment:
Unused vars?

Reply:
Thought aborted half way before I understood how django-su does things. Django-su has a weird way of switching back to the original user: by logging in to the original user without using a password. So, yes: you get the problem with the original su (and ssh for that matter): recursive impersonation.